testing/cli/test_cli_sct_maths.py

```diff
@@ -28,3 +32,17 @@ def test_sct_maths_add_images_no_checks():
     """Run the CLI script without checking results.
     TODO: Check the results. (This test replaces the 'sct_testing' test, which did not implement any checks.)"""
     sct_maths.main(argv=['-i', 'mt/mtr.nii.gz', '-add', 'mt/mtr.nii.gz', 'mt/mtr.nii.gz', '-o', 'test.nii.gz'])
+
+
+@pytest.mark.parametrize('dim', ['0', '1', '2'])
+def test_sct_maths_symmetrize(dim, tmp_path):
```

Comment:
Just to confirm, I tried this test on `master`, and it indeed fails for 2/3 of the dimensions. But on this branch, it passed for all 3 dimensions. So, looking good. :)

Reply:
great 😃